examples/twittermap/web/public/javascripts/map/countmap.js

```diff
@@ -141,7 +142,14 @@ angular.module('cloudberry.map')
       info.options = {
         position: 'topleft'
       };
-      $scope.controls.custom.push(info);
+      if ($scope.map){
+        $timeout(function(){
```

Comment:
What's the meaning of this timeout? I see we are adding the info here.

Reply:
I added the timeout to avoid some weird behavior as follow:

![mapbug](https://user-images.githubusercontent.com/27230373/39325843-18c18bc4-4948-11e8-90f7-b7357fa09446.jpg)

Every time I add the div back, it looks like this for a second before the correct text is shown. I think it is because it takes some time for the compile (line 138) to resolve the data. Right now I haven't found a better way to solve this problem.